flowmapper/cli.py

```diff
@@ -57,7 +59,11 @@ def map(
     """
     output_dir.mkdir(parents=True, exist_ok=True)
     field_mapping = read_field_mapping(fields)
-    source_flows = [Flow.from_dict(flow, field_mapping['source']) for flow in read_flowlist(source)]
+    source_flows = read_flowlist(source)
+    if transformations:
```

Comment:
There will be more than one migration file in some cases.

Reply:
Done in https://github.com/fjuniorr/flowmapper/pull/70/commits/e498de5637e933632ae61d3f227c9952a3a76f99.

I can't do something like:

```bash
flowmapper -t transformations.json migrations.json
```

because of [typer/click](https://github.com/tiangolo/typer/issues/110#issuecomment-643103250) so the call needs to be:

```bash
flowmapper -t transformations.json -t migrations.json
```